Recipes/TrainingDemo.recipe.json

```diff
@@ -65,6 +65,108 @@
           "Description": "The widgets in this layer are only displayed on the homepage."
         }
       ]
+    },
+    {
+      "name": "WorkflowType",
+      "data": [
+        {
+          "WorkflowTypeId": "47gk2q35xcbc0xg59kxwz5tkw4",
+          "Name": "LoginDateLogger",
+          "IsEnabled": true,
+          "IsSingleton": false,
+          "LockTimeout": 0,
+          "LockExpiration": 0,
+          "DeleteFinishedWorkflows": true,
+          "Activities": [
+            {
+              "ActivityId": "4w8zhbq9139e6zt4y2nb2gytdg",
+              "Name": "UserLoggedInEvent",
+              "X": 0,
+              "Y": 10,
+              "IsStart": true,
+              "Properties": {
+                "ActivityMetadata": {
+                  "Title": null
+                }
+              }
+            },
+            {
+              "ActivityId": "4bhq8xgddpzppy2j5m96q9whwf",
+              "Name": "LogTask",
+              "X": 320,
+              "Y": 0,
+              "IsStart": false,
+              "Properties": {
+                "ActivityMetadata": {
+                  "Title": null
+                },
+                "LogLevel": 3,
+                "Text": {
+                  "Expression": "{{ \"User logged in on\" | t }} {{ \"now\" | local | date: \"%c\" }}"
+                }
+              }
+            }
+          ],
+          "Transitions": [
+            {
+              "Id": 0,
+              "SourceActivityId": "4w8zhbq9139e6zt4y2nb2gytdg",
+              "SourceOutcomeName": "Done",
+              "DestinationActivityId": "4bhq8xgddpzppy2j5m96q9whwf"
+            }
+          ]
+        },
+        {
+          "WorkflowTypeId": "4vtt0g09pbnt8x72gnc6557smk",
+          "Name": "ContentCreatedLogTask",
+          "IsEnabled": true,
+          "IsSingleton": false,
+          "LockTimeout": 0,
+          "LockExpiration": 0,
+          "DeleteFinishedWorkflows": false,
+          "Activities": [
+            {
+              "ActivityId": "4jgzsrtras0wt5byehn5s9j04h",
+              "Name": "LogTask",
+              "X": 290,
+              "Y": 0,
+              "IsStart": false,
+              "Properties": {
+                "ActivityMetadata": {
+                  "Title": null
+                },
+                "LogLevel": 3,
+                "Text": {
+                  "Expression": "{{ User.Identity.Name }} added a content item"
```

Comment:
Itt lehet picit specifikusabb a bejegyzés, hiszen nem sima content itemet adott hozzá a user, hanem konkrétan Person Page-et. Mit gondolsz egy ilyesmiről?
"Expected non-error - {{ User.Identity.Name }} created a new Person Page content item."

Az Expected non-error benne van a többi helyen is, ahol logba írunk, pl.: DemoBackgroundTask.cs, azért tettem bele. Plusz ez jó ha van, hiszen ilyenkor a user látja, hogy ez a bejegyzés nem azért van itt, mert valami félrement, hanem csak simán reprezentációs célokat szolgál.

Illetve itt is használd a {{ | t }} liquid filtert, ahogy a másik esetben nagyon királyul használtad!

És akkor magának a Workflow-nak is át lehetne írni a nevét, hogy erre reflektáljon, például "Person Page created logger"